### PR TITLE
Week5

### PR DESCRIPTION
**Pull request by AmandaHockmuth**
Set up mongo, mongoose, and env. Refactored controller for createTask to integrate mongo. Tested with Postman. Confimed env and node_modules are excluded by gitignore.

### CODE REVIEW

File: 03-task-manager/starter/quizAnswers.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+(1) Quickly explain how middleware is used in an Express application.
+-- Middleware is/are functions automatically initialized when following a designated path.
```
Comment (jsuoz121) [inline 03-task-manager/starter/quizAnswers.txt:2]: Also worth noting that middleware in express runs in the context of a request, often between request receival and response generation, and is helpful for centralizing common operation

File: 03-task-manager/starter/package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "dotenv": "^8.2.0",
+    "dotenv": "^8.6.0",
```
Comment (jsuoz121) [inline 03-task-manager/starter/package.json:13]: Did you run into any issues with having 2 package.json files?

**Review by jsuoz121 — APPROVED** [review]
Nice job! The comments are informational, PR approved!

File: 03-task-manager/starter/package.json (hunk 2 above)
Comment (AmandaHockmuth) [reply 03-task-manager/starter/package.json:13]: I had not, how would I go about resolving that?

**Pull request merged**